Source/WebCore/loader/ResourceLoadInfo.cpp

```diff
@@ -152,18 +152,42 @@ std::optional<OptionSet<LoadContext>> readLoadContext(StringView name)
     return std::nullopt;
 }
 
+std::optional<OptionSet<RequestMethod>> readRequestMethod(StringView name)
```

Comment:
This would be `RequestMethod`, not `OptionSet` since only one method is used at a time.
```suggestion
std::optional<RequestMethod> readRequestMethod(StringView name)
```

Reply:
Is there any harm in leaving it as an `OptionSet`? It allows us to reuse the helper methods when parsing the JSON.